site/src/atomics/display.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+The following displays are covered: `block`, `flex`, `inline`, `inline-block`, `inline-flex`, `grid`, and `none`.
+
+The following breakpoints are supported for each display value:
```

Comment:
Re-arrange? On the current spacing documentation page (in PR) this example would be the last.

Reply:
Updated to match the new format in the spacing doc PR.